packages/playground/remote/src/lib/setup-fetch-network-transport.ts

```diff
@@ -84,7 +84,7 @@ export async function handleRequest(data: RequestData, fetchFn = fetch) {
 		response = await fetchFn(fetchUrl, {
 			method: fetchMethod,
 			headers: fetchHeaders,
-			body: data.data,
+			body: fetchMethod === 'GET' ? undefined : data.data,
```

Comment:
This is a tangential bugfix – I noticed some requests failed with "GET requests cannot have body" when `data.data` was an empty string.

Reply:
> This is a tangential bugfix – I noticed some requests failed with "GET requests cannot have body" when data.data was an empty string.

Nice.